core/src/main/scala/org/apache/spark/scheduler/EventLoggingListener.scala

```diff
@@ -255,9 +255,14 @@ private[spark] class EventLoggingListener(
     // ...
     // where jvmInformation, sparkProperties, etc. are sequence of tuples.
     // We go through the various  of properties and redact sensitive information from them.
-    val redactedProps = event.environmentDetails.map{ case (name, props) =>
-      name -> Utils.redact(sparkConf, props)
-    }
+    val noRedactProps = Seq("Classpath Entries")
+    val redactedProps = event.environmentDetails
+      .map {
```

Comment:
nit: This can be done like below, as they fit in one-line per each:

```
val redactedProps = event.environmentDetails.map {
  case (name, props) if noRedactProps.contains(name) => name -> props
  case (name, props) => name -> Utils.redact(sparkConf, props)
}
```

Reply:
Done. Modified as per the suggestion